docs/devguide/dtypes.rst

```diff
@@ -1,2 +1,77 @@
 Implementing a New Data Type
 ============================
+
+COMPAS data types are classes that are based on :class:`compas.data.Data`.
+
+Data types can be serialised to JSON with
```

Comment:
US-spelling police :P 

```suggestion
Data types can be serialized to JSON with
```

Reply:
and we like US spelling because...?

Reply:
I personally like US spelling better too... feels more common as a default